akka-contrib/src/test/scala/akka/contrib/circuitbreaker/sample/package.scala

```diff
@@ -0,0 +1,5 @@
+package akka.contrib.circuitbreaker
+
+package object sample {
+  case class AskFor(what: String)
+}
```

Comment:
please avoid package objects.
Let's put this in the Service actor's companion.

In docs it should also be explicit that this is the example Service's API, and not part of the circuit breaker API.